### DISCUSSION TITLE
how to save the signature in amazon s3 bucket in filament php

### DISCUSSION THREAD

**Discussion by abhi051002** [szimek/signature_pad#755] [Q&A]
` SignaturePad::make('signature')
                                ->backgroundColor('rgba(255,255,255,0)')
                                ->downloadable()
                                ->filename('autograph')
                                ->doneAction(fn (Action $action) => info("ddd"))
                                ->required()`
i used this code in filamentphp but can't figure out how to store the signature file in amazon s3 bucket

**Comment by UziTech** [comment]
You will probably get more answers if asked on stack overflow. You can create the data url by calling `signaturePad.toDataUrl()`.

**Reply by abhi051002** [reply]
but i'm working on filamentphp where this is not possible to use `signaturePad.toDataUrl()` because this is js code and i can't use it there

**Reply by UziTech** [reply]
So why are you asking on a JavaScript library if you can't use JavaScript?

signature_pad is only a JavaScript library

**Reply by abhi051002** [reply]
sorry for the inconvenience is there any way to do this in laravel?

**Reply by UziTech** [reply]
try https://www.google.com/search?q=laravel+signature+pad

**Reply by abhi051002** [reply]
thanks for the help let me try this package
